.github/ISSUE_TEMPLATE/suggestion.yml

```
@@ -0,0 +1,20 @@
+name: Suggestion
+description: Feel free to give us suggestions and enhancement ideas!
+labels: [enhancement]
+body:
+  - type: markdown
+    attributes:
+      value: "## Welcome!"
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking your time to give us suggestions! You gotta make sure these before reporting...
```

Wording can be improved here
```suggestion
        Thank you for taking your time to give us suggestions! Before opening an issue, please make sure that:
```